docs/sources/reference/components/otelcol/otelcol.receiver.syslog.md

```diff
@@ -0,0 +1,272 @@
+| tcp              | [tcp][]              | Configures a TCP syslog server to receive syslog messages.                                      | no*       |
+| tcp > multiline  | [multiline][]        | Configures rules for multiline parsing of incoming messages                                     | no       |
+| tcp > tls        | [tls][]              | Configures TLS for the TCP syslog server.                                                       | no       |
+| retry_on_failure | [retry_on_failure][] | Configures the retry behavior when the receiver encounters an error downstream in the pipeline. | no       |
+| debug_metrics    | [debug_metrics][]    | Configures the metrics that this component generates to monitor its state.                      | no       |
+| output           | [output][]           | Configures where to send received telemetry data.                                               | yes      |
+
+A syslog receiver must have either a `udp` or `tcp` block configured.
+
+The `>` symbol indicates deeper levels of nesting. For example, `tcp > tls`
+refers to a `tls` block defined inside a `tcp` block.
+
+[tls]: #tls-block
+[udp]: #udp-block
+[tcp]: #tcp-block
+[multiline]: #multiline-block
+[async]: #async-block
+[retry_on_failure]: #retry-on-failure-block
+[debug_metrics]: #debug_metrics-block
+[output]: #output-block
+
+### udp block
+
+The `udp` block configures a UDP syslog server.
+The following arguments are supported:
+
+| Name                            | Type     | Description                                                                                                  | Default | Required |
+|---------------------------------|----------|--------------------------------------------------------------------------------------------------------------|---------|----------|
+| `listen_address`                | `string` | The `<host:port>` address to listen to for syslog messages.                                                  |         | yes      |
+| `one_log_per_packet`            | `bool`   | Skip log tokenization, improving performance when messages always contain one log and multiline is not used. | `false` | no       |
+| `add_attributes`                | `bool`   | Add net.* attributes to log messages according to OpenTelemetry semantic conventions.                        | `false` | no       |
+| `encoding`                      | `string` | The encoding of the syslog messages.                                                                         | `utf-8` | no       |
+| `preserve_leading_whitespaces`  | `bool`   | Trims leading whitespace on messages unless this is set to `true`.                                           | `false` | no       |
+| `preserve_trailing_whitespaces` | `bool`   | Trims trailing whitespace on messages unless this is set to `true`.                                          | `false` | no       |
```

Comment:
I wonder... is the description for these two args backwards?

Would something like ``Preserves leading whitespace in messages when set to `true`.`` is a better description?